packages/@uppy/companion/src/standalone/index.js

```diff
@@ -181,16 +182,16 @@ if (app.get('env') === 'production') {
     // if the error is a URIError from the requested URL we only log the error message
     // to avoid uneccessary error alerts
     if (err.status === 400 && err instanceof URIError) {
-      console.error('\x1b[31m', req.id, err.message, '\x1b[0m')
+      logger.error(err.message, 'root.error', req.id)
     } else {
-      console.error('\x1b[31m', req.id, err, '\x1b[0m')
+      logger.error(err, 'root.error', req.id, true)
```

Comment:
this `true` argument is a little unclear to me, maybe `{ logStackTrace: true }` as an option is better?

Reply:
so I went more in favour of pulling it into a variable. The reason is, I feel like this issue may not necessarily exist when viewing from a code editor. This is because Typescript + JSDoc thankfully do well at annotating the function:

<img width="745" alt="Screenshot 2020-04-30 at 10 18 05" src="https://user-images.githubusercontent.com/8383781/80694213-1811c280-8acc-11ea-9a19-22cb8bb48235.png">

Reply:
i was more concerned about readability than writability, the github UI doesn't provide those signatures for example. i'm also OK with a named variable :)